iati/validator.py

```diff
@@ -445,6 +526,40 @@ def _parse_lxml_log_entry(log_entry):
     return error
 
 
+def _parse_ruleset_fail(rule):
```

Comment:
The name of this could be better... maybe something like `_create_error_for_rule`

Similarly, `_parse_lxml_log_entry` would get a new name along the lines of `_create_error_for_lxml_log_entry` - at the moment it's not exactly obvious what it is parsing to.

Reply:
Changed in 49c3f23a1b9d0896b32aef90c3d249ec54a72096